src/libultra/gu/cosf.c

```diff
@@ -1,6 +1,7 @@
 #include "ultra64.h"
 #include "libc/math.h"
-#include "macros.h"
+#include "ultra64.h"
+#include "z64math.h"
```

Comment:
Since these are libultra, I wonder if we should just not use the `SQ` macro and remove the z64math.h includes.